src/backend/marsha/core/api/video.py

```diff
@@ -594,12 +595,18 @@ def update_live_state(
         live_info.update(
             {"cloudwatch": {"logGroupName": serializer.validated_data["logGroupName"]}}
         )
-
         if serializer.validated_data["state"] == defaults.RUNNING:
             video.live_state = defaults.RUNNING
             live_info.update({"started_at": stamp})
             live_info.pop("stopped_at", None)
 
+            # if keys with no timeout were cached for this video, it needs to be reinitialized
+            key_cache_video = f"{defaults.VIDEO_ATTENDANCE_KEY_CACHE}{video.id}"
+            if list_keys := cache.get(key_cache_video, None):
+                for key in list_keys:
+                    cache.delete(key)
```

Comment:
You can't use `delete_many` to delete at once all the keys ? https://github.com/openfun/social-edu-federation/pull/12/files

Reply:
Yes i told you I would ...